docs/src/rules/quotes.md

```diff
@@ -15,14 +15,16 @@ var single = 'single';
 var backtick = `backtick`;    // ES6 only
 ```
 
-Each of these lines creates a string and, in some cases, can be used interchangeably. The choice of how to define strings in a codebase is a stylistic one outside of template literals (which allow embedded of expressions to be interpreted).
+Each of these lines creates a string and, in some cases, can be used interchangeably. The choice of how to define strings in a codebase is a stylistic one outside of template literals (which allow embedded expressions to be interpreted).
 
 Many codebases require strings to be defined in a consistent manner.
 
 ## Rule Details
 
 This rule enforces the consistent use of either backticks, double, or single quotes.
 
+This rule is aware of directive prologues such as `"use strict";` and will not flag or autofix them unless doing so will not change how the directive prologue is interpreted.
```

Comment:
A little difficult to parse, maybe:

```suggestion
This rule is aware of directive prologues such as `"use strict"` and will not flag or autofix them if doing so will change how the directive prologue is interpreted.
```

Reply:
Thanks! I thought that the semicolon in `"use strict";` would make it clearer that what is shown is a directive as opposed to just a string, but it that's bad for readability we can leave it out.

Reply:
Yeah, dangling semicolons are a little visually confusing. :)